package.json

```diff
@@ -31,13 +34,24 @@
   "scripts": {
     "build": "rollup -c",
     "dev": "rollup -c -w",
-    "test": "node test/test.js",
-    "pretest": "npm run build",
+    "test": "jest",
     "lint": "eslint --max-warnings 0 --ext .ts,js src/",
     "prettier": "prettier --write \"{src,example}/**/*.{ts,js}\"",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook"
   },
+  "jest": {
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js"
+    ],
+    "transform": {
+      "\\.(ts|tsx)$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
```

Comment:
jest doesn't like preprocessor (anymore)
```
(WARN) Replace any occurrences of "ts-jest/dist/preprocessor.js" or  "<rootDir>/node_modules/ts-jest/preprocessor.js" 
in the 'transform' section of your Jest config with just "ts-jest".
```

Reply:
Thank you, missed that! I have now also added `esModuleInterop` config option to suppress another warning, and fixed Axios interceptors (they expected `caches` to be defined which is not the case when running tests).